### PR TITLE
Slight change to ordered delivery guarantees

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

## Proposed Changes 

- slightly changing the doc for _ordered delivery guarantees_
-
-

### CODE REVIEW

File: docs/eventing/broker/kafka-broker/_index.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ spec:
 The supported consumer delivery guarantees are:
 
 * `unordered`: Unordered consumer is a non-blocking consumer that potentially delivers messages unordered, while preserving proper offset management.
-* `ordered`: Ordered consumer is a per-partition blocking consumer that delivers messages in order.
+* `ordered`: Ordered consumer is a per-partition blocking consumer, waiting for the Could Event receiver response before it delivers next messages of the partion.
```
Comment (pierDipi) [inline docs/eventing/broker/kafka-broker/_index.md:323]: typo: partition

Comment (matzew) [reply docs/eventing/broker/kafka-broker/_index.md]: @pierDipi thanks, I updated

**Review by pierDipi — APPROVED** [review]
/lgtm

File: docs/eventing/broker/kafka-broker/_index.md [hunk 2]
```diff
@@ -320,7 +320,7 @@ spec:
 The supported consumer delivery guarantees are:
 
 * `unordered`: Unordered consumer is a non-blocking consumer that potentially delivers messages unordered, while preserving proper offset management.
-* `ordered`: Ordered consumer is a per-partition blocking consumer that delivers messages in order.
+* `ordered`: Ordered consumer is a per-partition blocking consumer, waiting for the Could Event receiver response before it delivers next messages of the partition.
```
Comment (slinkydeveloper) [inline docs/eventing/broker/kafka-broker/_index.md:323]: s/Could Event/CloudEvent

s/receiver/subscriber

**Review by slinkydeveloper — CHANGES_REQUESTED** [review]

File: docs/eventing/broker/kafka-broker/_index.md [hunk 3]
```diff
@@ -320,7 +320,7 @@ spec:
 The supported consumer delivery guarantees are:
 
 * `unordered`: Unordered consumer is a non-blocking consumer that potentially delivers messages unordered, while preserving proper offset management.
-* `ordered`: Ordered consumer is a per-partition blocking consumer, waiting for the Could Event receiver response before it delivers next messages of the partition.
+* `ordered`: Ordered consumer is a per-partition blocking consumer, waiting for the CouldEvent subscriber response before it delivers next messages of the partition.
```
Comment (slinkydeveloper) [inline docs/eventing/broker/kafka-broker/_index.md:323]: s/CouldEvent/CloudEvent

**Review by slinkydeveloper — CHANGES_REQUESTED** [review]

**Review by slinkydeveloper — APPROVED** [review]

File: docs/eventing/broker/kafka-broker/_index.md [hunk 4]
```diff
@@ -320,7 +320,7 @@ spec:
 The supported consumer delivery guarantees are:
 
 * `unordered`: Unordered consumer is a non-blocking consumer that potentially delivers messages unordered, while preserving proper offset management.
-* `ordered`: Ordered consumer is a per-partition blocking consumer that delivers messages in order.
+* `ordered`: Ordered consumer is a per-partition blocking consumer, waiting for the CloudEvent subscriber response before it delivers next messages of the partition.
```
Comment (vaikas) [inline docs/eventing/broker/kafka-broker/_index.md:323]: Does it have to be successful response?

Comment (pierDipi) [reply docs/eventing/broker/kafka-broker/_index.md:323]: Great question/suggestion.

**Review by pierDipi — APPROVED** [review]

File: docs/eventing/broker/kafka-broker/_index.md [hunk 5]
```diff
@@ -320,7 +320,7 @@ spec:
 The supported consumer delivery guarantees are:
 
 * `unordered`: Unordered consumer is a non-blocking consumer that potentially delivers messages unordered, while preserving proper offset management.
-* `ordered`: Ordered consumer is a per-partition blocking consumer that delivers messages in order.
+* `ordered`: Ordered consumer is a per-partition blocking consumer, waiting for the CouldEvent subscriber's successful response before it delivers next messages of the partition.
```
Comment (vaikas) [inline docs/eventing/broker/kafka-broker/_index.md:323]: CouldEvent=>CloudEvent?

Comment (abrennan89) [inline docs/eventing/broker/kafka-broker/_index.md:323]:
```suggestion
* `ordered`: An ordered consumer is a per-partition blocking consumer that waits for a successful response from the CloudEvent subscriber before it delivers the next message of the partition.
```

File: docs/eventing/broker/kafka-broker/_index.md [hunk 6]
```diff
@@ -320,7 +320,7 @@ spec:
 The supported consumer delivery guarantees are:
 
 * `unordered`: Unordered consumer is a non-blocking consumer that potentially delivers messages unordered, while preserving proper offset management.
```
Comment (abrennan89) [inline docs/eventing/broker/kafka-broker/_index.md:322]:
```suggestion
* `unordered`:  An  unordered consumer is a non-blocking consumer that delivers messages unordered, while preserving proper offset management.
```

**Review by abrennan89 — CHANGES_REQUESTED** [review]
Suggestions

**Comment by matzew** [comment]
updated/rebased

**Review by abrennan89 — APPROVED** [review]

**Pull request merged**